web/src/ui_report.ts

```diff
@@ -55,7 +55,7 @@ export function success(response_html: string, $status_box: JQuery, remove_after
     message(response_html, $status_box, "alert-success", remove_after);
 }
 
-export function generic_embed_error(error_html: string, remove_after: number): void {
+export function generic_embed_error(error_html: string, remove_after: number | undefined): void {
```

Comment:
Use `remove_after?: number` (documentation: [optional parameters](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters)) so that the caller doesn’t need to explicitly passed `undefined`.